src/Umbraco.Web/PropertyEditors/ContentPickerPropertyEditor.cs

```diff
@@ -33,17 +33,12 @@ protected override PreValueEditor CreatePreValueEditor()
 
         internal class ContentPickerPreValueEditor : PreValueEditor
         {
-            [PreValueField("showOpenButton", "Show open button", "boolean")]
+            [PreValueField("showOpenButton", "Show open button (this feature is in preview!)", "boolean", Description = " Opens the node in a dialog")]

```

Comment:
Currently looking at C# core codebase to see if we can provide a UI translation key as opposed to a hardcoded english string. Will update you with my findings shortly